examples/speaker_tasks/diarization/neural_diarizer/e2e_diarize_speech.py

```diff
@@ -0,0 +1,434 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+Usage:
+End-to-end speaker diarization model can be specified by either "model_path" or "pretrained_name".
```

Comment:
Shall we add more explanation on whether this script is for evaluation (e.g., calculating DER and thus requires RTTM labels in manifest) or is is for inference (diarize any input speech)? If both, how to setup the script for different cases?